chain/beacon.go

```diff
@@ -68,9 +69,29 @@ func (b *Beacon) String() string {
 }
 
 func shortSigStr(sig []byte) string {
+	if sig == nil {
```

Comment:
this scares me haha

Reply:
It can happen for the unchained network. It's weird that it hasn't failed so far.